crates/iota-types/src/scoring_metrics.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::atomic::{AtomicU64, Ordering};
+
+use iota_protocol_config::ProtocolConfig;
+
+use crate::messages_consensus::{ReportV1, VersionedReport};
+
+pub enum ScoringMetrics {
+    V1(ScoringMetricsV1),
+}
+
+pub struct ScoringMetricsV1 {
```

Comment:
Comment needed explaining what the intended usage of this is.

Reply:
Perhaps a name to indicate that this represents a single authority viewpoint, such as `LocalScoringMetrics` would be helpful.

Reply:
I kept the name of the structure as it is, because it is not only used for local metrics counts, it is something more generic. But I agree that there is room for improvement in all these names, so I'm adding some aliases so anything involving those types are easier to understand.